src/appengine/handlers/cron/cleanup.py

```diff
@@ -338,14 +338,14 @@ def get_issue_for_testcase(testcase):
   if not testcase.bug_information:
     return None
 
-  issue_tracker_manager = issue_tracker_utils.get_issue_tracker_manager(
-      testcase=testcase, use_cache=True)
-  if not issue_tracker_manager:
+  issue_tracker = issue_tracker_utils.get_issue_tracker_for_testcase(
+      testcase, use_cache=True)
+  if not issue_tracker:
     return None
 
   try:
-    issue_id = int(testcase.bug_information)
-    issue = issue_tracker_manager.get_original_issue(issue_id)
+    issue_id = testcase.bug_information
```

Comment:
Isn't testcase.bug_information a string, should get_issue and get_original_issue do the int conversion automatically ?

Reply:
Yes get_issue and get_original_issue should handle the int conversion automatically. No need for callers to convert to int everywhere.